qemu/tests/hotplug_mem_share_discard_data.py

```diff
@@ -0,0 +1,97 @@
+from avocado.core import exceptions
+from avocado.utils import process
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_misc
+
+from virttest.utils_numeric import normalize_data_size
+from virttest.utils_test.qemu import MemoryHotplugTest
+from virttest.staging import utils_memory
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Memory share and discard-data hotplug test
+    1) Setup hugepages
+    2) Mount /mnt/kvm_hugepage
+    3) Boot guest with one numa node
+    4) Hotplug 1G memory
+    5) Check memory
+    6) Unplug memory
+    7) Check memory
+    8) Check results for discard-data value
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    timeout = float(params.get("login_timeout", 240))
+
+    vm = params['main_vm']
+    env_process.preprocess_vm(test, params, env, vm)
```

Comment:
L31~L34 can be removed,  and the cfg 
```
    start_vm = yes
    not_preprocess = yes
```
can be removed too, let framework launch VM directly.

Reply:
Changed!